packages/remix-dev/config.ts

```diff
@@ -268,6 +278,17 @@ export interface RemixConfig {
    */
   assetsBuildDirectory: string;
 
+  /**
+   * The absolute path to the metafile build directory.
+   */
+  debugDirectory: string
+
+  /**
+   * Whether to support generate browser js metafile for esbuild, default to false.
+   * When you set it to true, remix-js-metafile.json will be generated in the assetsBuildDirectory directory.
+   */
+  metafile?: boolean;
```

Comment:
`metafile` should be specifiable as a config option as you have here, but also as a flag `--metafile` for both `remix build` and `remix dev`.

Reply:
Currently, metafile in my code will also be enabled according to the configuration when you use remix dev. But actually, I think maybe it's a good idea to shut down generate metafile all the time under remix dev.